dnf5/commands/history/history_store.cpp

```diff
@@ -55,11 +55,13 @@ void HistoryStoreCommand::run() {
     std::vector<libdnf5::transaction::Transaction> transactions;
     auto logger = get_context().get_base().get_logger();
 
-    std::filesystem::path tmp_path(output_option->get_value());
+    std::filesystem::create_directories(output_option->get_value());
 
-    if (std::filesystem::exists(tmp_path)) {
+    std::filesystem::path trans_file_path(output_option->get_value() + TRANSACTION_JSON);
```

Comment:
I believe that this is correct - `output_option->get_value() / TRANSACTION_JSON`